hawkbit-repository/hawkbit-repository-jpa/src/test/java/org/eclipse/hawkbit/repository/jpa/ActionStatusUpdateHandlerServiceTest.java

```diff
@@ -0,0 +1,118 @@
+package org.eclipse.hawkbit.repository.jpa;
```

Comment:
Please provide a proper licence for this file

Reply:
license note added